src/mgr/MgrClient.cc

```diff
@@ -286,11 +280,6 @@ int MgrClient::start_command(const vector<string>& cmd, const bufferlist& inbl,
 
   ldout(cct, 20) << "cmd: " << cmd << dendl;
 
-  if (!session) {
-    lderr(cct) << "no session, waiting" << dendl;
-    wait_on_list(waiting_for_session);
-  }
```

Comment:
`CommandTable` queues the commands up, and a command is removed from queue only when it is acked by `handle_command_reply()`.